plugins/org.eclipse.n4js.tester/src/org/eclipse/n4js/tester/TesterModule.java

```diff
@@ -54,27 +56,61 @@
 import org.eclipse.n4js.tester.internal.Utf8UrlDecoderService;
 import org.eclipse.n4js.tester.server.HttpServerManager;
 import org.eclipse.n4js.tester.server.JettyManager;
+import org.eclipse.n4js.utils.ContainerTypesHelper;
+import org.eclipse.n4js.utils.ResourceNameComputer;
 
-import com.google.inject.AbstractModule;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.inject.Binder;
+import com.google.inject.Injector;
+import com.google.inject.Module;
+import com.google.inject.name.Names;
 
 /**
- * Module for the N4 tester core component.
+ * Defines bindings to N4JS instances in the ui case.
  */
-public class TesterModule extends AbstractModule {
-
+public class TesterModule implements Module {
 	private static final Logger LOGGER = getLogger(TesterModule.class);
 
+	final private Injector n4jsInjector;
+
+	/** Called in the ui case */
+	public TesterModule(Injector n4jsInjector) {
+		this.n4jsInjector = n4jsInjector;
```

Comment:
In general this might not be N4JS injector, just injector providing bindings for some types in `n4js` (non-UI).